worker/worker.py

```diff
@@ -234,7 +234,6 @@ def main():
   success = True
   global ALIVE
   while ALIVE:
-    print(path.join(worker_dir, 'fish.exit'))
     if path.isfile(worker_dir, 'fish.exit'):
```

Comment:
if path.isfile(path.join(worker_dir, 'fish.exit')):

Reply:
@tomtor I beg your pardon for wasting your time. Wrong copy&past, editing w/o , comments too short. I promise to review the code only if having a proper time and not during a multitasking frenzy...